content/sensu-go/5.15/release-notes.md

```diff
@@ -7,6 +7,8 @@ version: "5.15"
 menu: "sensu-go-5.15"
 ---
 
+- [5.15.0 release notes](#5-15-0-release-notes)
+- [5.14.1 release notes](#5-14-1-release-notes)
```

Comment:
Did we not have release notes for 5.14.2?

Reply:
Yes, I need to add them. Thank you for catching it.